de.metas.adempiere.adempiere/base/src/main/java-legacy/org/adempiere/model/RelationTypeZoomProvider.java

```diff
@@ -384,6 +435,12 @@ public int getAD_Window_ID(final boolean isSOTrx)
 
 		public boolean matchesAsSource(final IZoomSource zoomSource)
 		{
+			if(isReferenceTarget())
+			{
+				// the source always matches if the target is ReferenceTarget
```

Comment:
this comment would be great as part of the method's javadoc

Reply:
I also improved the java doc. Please, ping me if I lost something.